spec/classes/init_spec.rb

```diff
@@ -2045,6 +2047,25 @@
     end
   end
 
+  context 'with sshd_config_maxauthtries not being a valid number' do
```

Comment:
Ok or maybe something like :
validate_re($sshd_config_maxauthtries, '^\d+$', "ssh::sshd_config_maxauthtries must be a valid number and is set to <${sshd_config_maxauthtries}>.")
Thanks you.